streaming/src/main/java/org/openmrs/analytics/FhirEventWorker.java

```diff
@@ -92,15 +84,17 @@ public void process(Event event) {
 				log.info("Skipping non-FHIR event " + event);
 				return;
 			}
+			
+			if (!fhirUrl.contains("fhir2"))
+				fhirUrl = fhirUrl.replace("/fhir", "/fhir2/R4");
```

Comment:
Hmm, why do you need this logic? The assumption is that any such changes should be done on the server side. When you add the AtomFeed module did you also update the config as described [here](https://github.com/GoogleCloudPlatform/openmrs-fhir-analytics#add-fhir2-module-to-openmrs-and-update-atom-feed-config)?

Reply:
I did run the configuration, but was at some point having weird issues with the urls. 

This is mostly another vestige of the iSantePlus testing, and is not needed anymore. I did run into other issues with urls that were being saved in the `markers` table by the atomfeed client, and had to do the following fixes to work with https://isanteplusdemo.com/openmrs: 
- https://github.com/pmanko/atomfeed/blob/5a44e0ce1fded48ffe96808f8c31d06b5ee87419/atomfeed-client/src/main/java/org/ict4h/atomfeed/client/service/FeedEnumerator.java#L44
- https://github.com/pmanko/atomfeed/blob/5a44e0ce1fded48ffe96808f8c31d06b5ee87419/atomfeed-client/src/main/java/org/ict4h/atomfeed/client/service/FeedEnumerator.java#L92

Reply:
So do you want to drop this change from here or should we keep it? I still think this should be handled at the config file level.